python3-cryptography.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "python3-cryptography",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"cryptography\" --no-build-isolation"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/68/ce/95b0bae7968c65473e1298efb042e10cafc7bafc14d9e4f154008241c91d/cffi-1.16.0.tar.gz",
+            "sha256": "bcb3ef43e58665bbda2fb198698fcae6776483e0c4a631aa5647806c25e02cc0"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/85/62/48bcebd955945d8da3fe9b84a679dbf4bf179e1ac36e583b7eaa47506758/cryptography-41.0.5-cp37-abi3-manylinux_2_28_x86_64.whl",
```

Comment:
This is x86_64 only you should either use a source tarball from pypi or add aarch64 binary wheel from pypi and add only-arch

Reply:
A source build requires this:

https://github.com/flathub/io.github.hydrusnetwork.hydrus/blob/ce51152994cd3beb6e240fda31711d6d777d2150/python3-sources.json#L572-L641

https://github.com/flathub/io.github.hydrusnetwork.hydrus/blob/ce51152994cd3beb6e240fda31711d6d777d2150/io.github.hydrusnetwork.hydrus.yml#L6-L7 

and a cargo-sources.json file for cryptography. Extract the tarball from pypi, go into src/rust then use https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo on the Cargo.lock file there.

Reply:
Cryptography is so cumbersome that this software has removed the cryptography dependency now